installer-resources/chained.yaml

```diff
@@ -1,3 +1,4 @@
+chainedservers:
```

Comment:
We don't actually use the checked in copy - the copy we built with has that
line.
On Sep 9, 2015 8:25 PM, "fffw" <notifications@github.com> wrote:

> In installer-resources/chained.yaml
> <https://github.com/getlantern/lantern/pull/3121#discussion_r39121457>:
>
> > @@ -1,3 +1,4 @@
> > +chainedservers:
>
> Without this line, len(servers.ChainedServers) will always be zero when
> reading from chained.yaml
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/pull/3121/files#r39121457>.
>